selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/utils/BrowserLogsUtil.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.selenium.core.utils;
+
+import static java.lang.String.format;
+import static org.openqa.selenium.logging.LogType.BROWSER;
+import static org.openqa.selenium.logging.LogType.CLIENT;
+import static org.openqa.selenium.logging.LogType.DRIVER;
+import static org.openqa.selenium.logging.LogType.PERFORMANCE;
+import static org.openqa.selenium.logging.LogType.PROFILER;
+import static org.openqa.selenium.logging.LogType.SERVER;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import java.util.Collections;
+import java.util.List;
+import org.eclipse.che.selenium.core.SeleniumWebDriver;
+import org.openqa.selenium.logging.LogEntry;
+import org.openqa.selenium.logging.LogType;
+import org.openqa.selenium.logging.Logs;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Singleton
+public class BrowserLogsUtil {
+  private final SeleniumWebDriver seleniumWebDriver;
+  private final Logger LOG = LoggerFactory.getLogger(BrowserLogsUtil.class);
+
+  @Inject
+  public BrowserLogsUtil(SeleniumWebDriver seleniumWebDriver) {
+    this.seleniumWebDriver = seleniumWebDriver;
+  }
+
+  /**
+   * read logs of provided type
+   *
+   * @param logType
+   * @see LogType
+   * @return provided type of browser logs
+   */
+  public List<LogEntry> getLogs(String logType) {
+    Logs logs = seleniumWebDriver.manage().logs();
+    List<LogEntry> result = Collections.emptyList();
```

Comment:
**result** field is redundant if we refactor code as mentioned [here](https://github.com/eclipse/che/pull/8502/files#r165030896).